### ISSUE TITLE
Error when the path to a playbook contains a space

### ISSUE THREAD

**Issue by MiLk** [willthames/ansible-lint#62]
If the path to my playbook contains a space, it fails to open it.

The culprit seems to be https://github.com/willthames/ansible-lint/blob/master/lib/ansiblelint/utils.py#L93

```bash
$ ./venv/bin/ansible-lint /Users/x/Library/Application Support/x/x/playbook.yml
Traceback (most recent call last):
  File "./venv/bin/ansible-lint", line 110, in <module>
    sys.exit(main(sys.argv[1:]))
  File "./venv/bin/ansible-lint", line 97, in main
    matches = runner.run()
  File "/Users/x/x/x/venv/lib/python2.7/site-packages/ansiblelint/__init__.py", line 178, in run
    skip_tags=set(self.skip_tags)))
  File "/Users/x/x/x/venv/lib/python2.7/site-packages/ansiblelint/__init__.py", line 106, in run
    with open(playbookfile['path'], 'Ur') as f:
IOError: [Errno 2] No such file or directory: 'Support/x/x/playbook.yml'
```

**Comment by wybczu** [comment]
You are simply passing two arguments to ansible-lint. Try

```
$ ./venv/bin/ansible-lint '/Users/x/Library/Application Support/x/x/playbook.yml'
```

or

```
$ ./venv/bin/ansible-lint /Users/x/Library/Application\ Support/x/x/playbook.yml
```

**Comment by MiLk** [comment]
Indeed, I copied the wrong test result.

```
$ ./venv/bin/ansible-lint '/Users/x/Library/Application Support/x/x/playbook.yml'
Traceback (most recent call last):
  File "./venv/bin/ansible-lint", line 110, in <module>
    sys.exit(main(sys.argv[1:]))
  File "./venv/bin/ansible-lint", line 97, in main
    matches = runner.run()
  File "/Users/x/x/x/venv/lib/python2.7/site-packages/ansiblelint/__init__.py", line 178, in run
    skip_tags=set(self.skip_tags)))
  File "/Users/x/x/x/venv/lib/python2.7/site-packages/ansiblelint/__init__.py", line 106, in run
    with open(playbookfile['path'], 'Ur') as f:
IOError: [Errno 2] No such file or directory: '/Users/x/Library/Application'
```

```
$ ./venv/bin/ansible-lint /Users/x/Library/Application\ Support/x/x/playbook.yml
Traceback (most recent call last):
  File "./venv/bin/ansible-lint", line 110, in <module>
    sys.exit(main(sys.argv[1:]))
  File "./venv/bin/ansible-lint", line 97, in main
    matches = runner.run()
  File "/Users/x/x/x/venv/lib/python2.7/site-packages/ansiblelint/__init__.py", line 178, in run
    skip_tags=set(self.skip_tags)))
  File "/Users/x/x/x/venv/lib/python2.7/site-packages/ansiblelint/__init__.py", line 106, in run
    with open(playbookfile['path'], 'Ur') as f:
IOError: [Errno 2] No such file or directory: '/Users/x/Library/Application'
```

**Comment by willthames** [comment]
Fixed by #63
